lib/processors/processor.dart

```diff
@@ -76,6 +77,9 @@ class Processor extends AbstractProcessor<void> {
     'ios:plist',
     'ios:launchScreen',
 
+    // IDE
+    'ide:vscode',
```

Comment:
I would remove it from the default instructions set since not everyone uses vscode.

Reply:
How do you feel about it being part of the config?:
```yaml
flavorizr:
  app:
    ...
  flavors:
    apple:
      ...
    banana:
      ...
  ide: vscode
```

Reply:
I was exactly thinking about that. In that case, it should look similar to the dummyasset processor. I would consider it optional and I would use an enum. If that field is missing, then no ide processor is executed.